eZ/Bundle/EzPublishCoreBundle/Command/RemoveVersionsCommand.php

```diff
@@ -0,0 +1,198 @@
+use Symfony\Component\Console\Output\Output;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class RemoveVersionsCommand extends Command
+{
+    const DEFAULT_REPOSITORY_USER = 'admin';
+    const DEFAULT_KEEP = 2;
+
+    const VERSION_DRAFT = 'draft';
+    const VERSION_ARCHIVED = 'archived';
+    const VERSION_ALL = 'all';
+
+    /**
+     * @var \eZ\Publish\API\Repository\UserService
+     */
+    private $userService;
+
+    /**
+     * @var \eZ\Publish\API\Repository\ContentService
+     */
+    private $contentService;
+
+    /**
+     * @var \eZ\Publish\API\Repository\PermissionResolver
+     */
+    private $permissionResolver;
+
+    /**
+     * @var \Doctrine\DBAL\Driver\Connection
+     */
+    private $connection;
+
+    public function __construct(
+        UserService $userService,
+        ContentService $contentService,
+        PermissionResolver $permissionResolver,
+        Connection $connection
+    ) {
+        $this->userService = $userService;
+        $this->contentService = $contentService;
+        $this->permissionResolver = $permissionResolver;
+        $this->connection = $connection;
+
+        parent::__construct();
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        parent::initialize($input, $output);
+
+        $this->permissionResolver->setCurrentUserReference(
+            $this->userService->loadUserByLogin($input->getOption('user'))
+        );
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('ezplatform:remove-versions')
+            ->setDescription('Remove unwanted content versions. It keeps published version untouched. By default, it keeps also the last archived/draft version.')
+            ->addOption(
+                'status',
+                's',
+                InputOption::VALUE_OPTIONAL,
+                sprintf(
+                    "Select which version types should be removed: '%s', '%s', '%s'.",
+                    self::VERSION_DRAFT,
+                    self::VERSION_ARCHIVED,
+                    self::VERSION_ALL
+                ),
+                self::VERSION_ALL
+            )
+            ->addOption(
+                'keep',
+                'k',
+                InputOption::VALUE_OPTIONAL,
+                "Sets number of the most recent versions (both drafts and archived) which won't be removed.",
+            self::DEFAULT_KEEP
+            )
+            ->addOption(
+                'user',
+                'u',
+                InputOption::VALUE_OPTIONAL,
+                'eZ Platform username (with Role containing at least Content policies: remove, read, versionread)',
+                self::DEFAULT_REPOSITORY_USER
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $contentIds = $this->getObjectsIds((int) $input->getOption('keep'), $input->getOption('status'));
```

Comment:
You are using `(int) $input->getOption('keep')` and `$input->getOption('status')` expression multiple times. Could you extract value of them to variables ?